### ISSUE TITLE
Atom using a lot of CPU

### ISSUE THREAD

**Issue by cquiroz** [laughedelic/atom-ide-scala#25]
Sometimes I see atom using a high amount of CPU which i think it's related to this plugin. How could I confirm for sure?

Here's an example
![screenshot 2017-12-18 17 13 54](https://user-images.githubusercontent.com/3615303/34126056-134f27be-e417-11e7-9cbb-77290c19a455.png)

**Comment by laughedelic** [comment]
That's odd. I haven't seen it so far. I doubt that the plugin itself causes this because it just wires the language server to the atom-ide-ui using atom-languageclient. If this was caused by the server, I guess you would see the load on `java` process. It could be caused by the atom-languageclient or some other unrelated Atom-specific thing...

Have you noticed any pattern in the situations when it happens? Any specific (big) projects or does it happen when you do certain actions? 

You could take a look at the logs (check the docs [here](https://github.com/laughedelic/atom-ide-scala/blob/master/DEVELOPMENT.md#debugging)), both inside Atom and in the file from the server and see if there is anything suspicious when it happens.

**Comment by cquiroz** [comment]
Right, it's difficult to say if this is related to `atom-ide-scala` or some other plugin. I saw it only once today with a single scala project open but I can't tell a specific action that produced it. I'll keep my eyes open if it happens again

**Comment by cquiroz** [comment]
I have seen this behaviour which directly affects CPU usage. i see that multiple language servers are being launched. i just do normall editing of files and use sbt in triggered mode for compilation and after a while I see multiple instances, e.g.

```
 3729 93333 93331   0 12:21PM ??        11:51.64 /usr/local/opt/jenv/versions/1.8/bin/java -jar /Users/cquiroz/.atom/packages/ide-scala/coursier launch --quiet --extra-jars /usr/local/opt/jenv/versions/1.8/lib/tools.jar --repository bintray:dhpcs/maven --repository bintray:scalameta/maven org.scalameta:metaserver_2.12:0.1-SNAPSHOT --main scala.meta.languageserver.Main
 3729 94440 94438   0 12:32PM ??         0:20.56 /usr/local/opt/jenv/versions/1.8/bin/java -jar /Users/cquiroz/.atom/packages/ide-scala/coursier launch --quiet --extra-jars /usr/local/opt/jenv/versions/1.8/lib/tools.jar --repository bintray:dhpcs/maven --repository bintray:scalameta/maven org.scalameta:metaserver_2.12:0.1-SNAPSHOT --main scala.meta.languageserver.Main
```

If I kill them and restart atom CPU usage returns to normal
